types/ember/index.d.ts

```diff
@@ -18,6 +18,18 @@ declare module 'ember' {
     import Rsvp from 'rsvp';
     import { TemplateFactory } from 'htmlbars-inline-precompile';
 
+    // A type registry for Ember `Service`s. Meant to be declaration-merged so
+    // string lookups resolve to the correct type.
+    export interface ServiceRegistry {
+        [serviceName: string]: Ember.Service;
```

Comment:
Are there default services we should include, like the router service? The ember-data should also declaration merge the `'store'` service into here

Reply:
Oh, yes, good point. I thought this included the `'store'` injection already, but forgot to push that. I’ll confirm whether or not there are any others besides the store and the router and push a change with them.